tx_list_test.go

```diff
@@ -186,11 +186,12 @@ func TestTx_LPop(t *testing.T) {
 	runNutsDBTest(t, nil, func(t *testing.T, db *DB) {
 		txCreateBucket(t, db, DataStructureList, bucket, nil)
 		txPop(t, db, bucket, GetTestBytes(0), nil, ErrListNotFound, true)
+
+		db.Close()
```

Comment:
On windows, I can run the next case if we don't close db in here. The files will be occupated. Some other unit cases same with here.